client/src/components/Sidebar/ChatContent.js

```diff
@@ -19,26 +20,27 @@ const useStyles = makeStyles((theme) => ({
     letterSpacing: -0.17,
   },
   notification: {
-    height: 20,
-    width: 20,
     backgroundColor: "#3F92FF",
-    marginRight: 10,
+    marginRight: 20,
     color: "white",
     fontSize: 10,
-    letterSpacing: -0.5,
     fontWeight: "bold",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    borderRadius: 10,
   },
 }));
 
 const ChatContent = (props) => {
   const classes = useStyles();
 
-  const { conversation } = props;
-  const { latestMessageText, otherUser } = conversation;
+  const { conversation, activeConversation } = props;
+  const { latestMessageText, otherUser, messages } = conversation;
+
+  //get all unread message for all chats except the current active user
+  const unReadMessages = messages.filter(
+    (message) =>
+      message.isRead === false &&
+      message.senderId === otherUser.id &&
+      activeConversation != otherUser.username
+  ).length;
```

Comment:
Again, there are likely more efficient solutions for this counting, potentially on the backend.

Reply:
I added the count in the backend